internal/dinosaur/pkg/cmd/fleetmanagerclient/client.go

```diff
@@ -0,0 +1,60 @@
+// Package fleetmanagerclient is a client for the CLI to connect to the fleetmanager.
+package fleetmanagerclient
+
+import (
+	"os"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager"
+)
+
+var (
+	singletonInstance sync.Once
+	client            *fleetmanager.Client
+)
+
+const (
+	defaultFleetManagerEndpoint = "http://localhost:8000"
+	fleetManagerEndpointEnvVar  = "FMCLI_FLEET_MANAGER_ENDPOINT"
+	ocmRefreshTokenEnvVar       = "OCM_TOKEN"
+)
+
+// AuthenticatedClientWithOCM returns a rest client to the fleet-manager and receives the OCM refresh token.
+// This function will panic on an error, designed to be used by the fleet-manager CLI.
+func AuthenticatedClientWithOCM() *fleetmanager.Client {
+	if os.Getenv(ocmRefreshTokenEnvVar) == "" {
```

Comment:
should this also be extracted to a local var ?
```suggestion
        ocmRefreshToken := os.Getenv(ocmRefreshTokenEnvVar)
	if ocmRefreshToken == "" {
```

Reply:
done